projects/components/src/table/cells/data-renderers/string-array/string-array-table-cell-renderer.component.ts

```diff
@@ -0,0 +1,60 @@
+import { ChangeDetectionStrategy, Component, Inject, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { TableColumnConfig, TableRow } from '../../../table-api';
+import {
+  TABLE_CELL_DATA,
+  TABLE_COLUMN_CONFIG,
+  TABLE_COLUMN_INDEX,
+  TABLE_DATA_PARSER,
+  TABLE_ROW_DATA
+} from '../../table-cell-injection';
+import { TableCellParserBase } from '../../table-cell-parser-base';
+import { TableCellRenderer } from '../../table-cell-renderer';
+import { TableCellRendererBase } from '../../table-cell-renderer-base';
+import { CoreTableCellParserType } from '../../types/core-table-cell-parser-type';
+import { CoreTableCellRendererType } from '../../types/core-table-cell-renderer-type';
+import { TableCellAlignmentType } from '../../types/table-cell-alignment-type';
+
+@Component({
+  selector: 'ht-string-array-table-cell-renderer',
+  styleUrls: ['./string-array-table-cell-renderer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="string-array-cell">
+      <span class="first-item">{{ this.firstItem }}</span>
+      <span class="summary-text" [htTooltip]="this.summaryTooltip">{{ this.summaryText }}</span>
```

Comment:
So I think the tooltip should be across the whole cell, and show all values. This actually just came up when @anandtiwary implemented basically exactly this component... can we share an underlying component? let me go find it

Reply:
Linked in the summary comment

Reply:
@aaron-steinfeld the link you used is for this PR. :)

Reply:
I don't think so? Linked to #615, this is #655 no matter how similar they might look :)

Reply:
Sorry. My browser was not redirecting when I clicked on the link. So I assumed its this page. I'll check that PR.

Reply:
@aaron-steinfeld Checked that PR. I'm not sure if we should use that here. #655 adds them as summary values which has a certain format. If there are no icons present, it shows a dot. To circumvent that usecase we'll have to modify the summary value component's behaviour. Also override fonts etc. I don't think we should do that just to reuse it. Our usecase here is plain string array display. 

We cannot take this out into a component and have it used there either because that component is built for summary values usecase. The API is more complex with optional icons, labels etc.

One thing we can do is to write a wrapper which covers both cases that'd use summary value / plain text based on requirement. But then we'd be writing a component with a pretty confusing/complicated API to share one template and couple lines of code.

Reply:
OK - definitely not worth maintaining that level of difference in one component. I just wish we'd stop using similar but slightly unique components everywhere...

Reply:
We should really consolidate the two styles. I think per michael it is the same component. This would lead to inconsistency, right?

Reply:
I'll file a tech debt for this.